src/Features/Navigation/Menu.php

```diff
@@ -513,6 +513,54 @@ public function add_core_items() {
 		}
 	}
 
+	/**
+	 * Add an item or taxonomy.
+	 *
+	 * @param array $menu_item Menu item.
+	 */
+	public function add_item_and_taxonomy( $menu_item ) {
+		if ( in_array( $menu_item[2], CoreMenu::get_excluded_items(), true ) ) {
```

Comment:
You reference `$menu_item[2]` several times here. Totally optional, but I might assign this to a temporary variable with a descriptive name.